src/scripts/OutSystems/OSUI/ErrorCodes.ts

```diff
@@ -311,6 +311,13 @@ namespace OutSystems.OSUI.ErrorCodes {
 		FailRegisterCallback: 'OSUI-API-29003',
 	};
 
+	export const Video = {
+		FailChangeProperty: 'OSUI-API-30001',
```

Comment:
this will create a conflict with the ones I created on OverflowMenu. Can you change to 31001, etc?

Reply:
Yes, I will change it.